payments/trustly/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Swedbank Pay Trustly Payments
+sidebar:
+  navigation:
+  - title: Trustly Payments
+    items:
+    - url: /payments/trustly
+      title: Introduction
+    - url: /payments/trustly/redirect
+      title: Redirect
+    - url: /payments/trustly/seamless-view
+      title: Seamless View
+    - url: /payments/trustly/after-payment
+      title: After Payment
+    - url: /payments/trustly/other-features
+      title: Other Features
+---
+
+{% include jumbotron.html body="**Trustly Payments** is one of the easiest
+                                payment service where Swedbank Pay helps improve
+                                cashflow by purchasing merchant invoices. Choose
+                                between our **Redirect** and
+                                **Seamless view** integration options." %}
+
+{% include alert.html type="informative"
+                      icon="cached"
+                      body="**Redirect** is the easiest way to implement Invoice
+                      Payments. Redirect will take your consumer to a Swedbank
+                      Pay hosted payment page where they can perform a secure
+                      transaction. The consumer will be redirected back to your
+                      website after the completion of the payment." %}
+
+{% include alert.html type="informative"
+                      icon="branding_watermark"
+                      body="**Seamless View** is our solution for a payment
+                      experience that is integrated directly on your website.
+                      The payment process will be executed in an `iframe` on
+                      your page." %}
+
+### Important steps before you launch Swedbank Pay Invoice Payments at your website
+
+Prior to launching Trustly Payments at your site, make sure that
+you have done the following:
+
+1. Make sure that the payment option is displayed with a Trustly logo.
+
+## API Requests
+
+The API requests are displayed in the sales flow below.
+
+## Languages
+
+Trustly is supported in Swedish.
+
+## Trustly flow
+
+This is an example of the Redirect scenario. For other integrations, take a
+look at the respective sections. The sequence diagram below shows the two
+requests you have to send to Swedbank Pay to make a purchase. The diagram also
+shows the steps in a [sale][sale] process.
+
+The Trustly integration uses the Deposit call by performing a payment. After
+SwedbankPay perform a `Deposit` call https://trustly.com/en/developer/api#/deposit, the end-user will be presented with the returned IFrame URL in order to perform
+the payment with their prefered bank. Once the user has completed the payment,
+SwedbankPay will receive a notification asynchronously from Trustly, hence why
+the UI will initiate polling toward our back-end. The payment status after being
+redirect to `completeUrl` will then indicate if the payment was successful or
+not, or if the payment is still in progress.
```

Comment:
Unsure if we should expose this complexity in our documentation. I feel it is relevant for how the overall payment process works, but it is not required to be fully understood by a service integrator. What do you guys think? @asbjornu @marinafrydenb